data_structures/queue_using_one_stack.cpp

```diff
@@ -0,0 +1,122 @@
+/*
+  @author [Rajat Garg](https://github.com/Rajat379)
+  @file
+
+  Implementation of a Queue using one Stack.
+*/
+
+#include <cassert>
+#include <iostream>
+#include <stack>
+
+namespace {
```

Comment:
Namespaces should usually have one.
```suggestion
/**
 * @brief Data structures
 * @namespace data_structures
 */
namespace data_structures {
```